docs/guides/testing/testing.rst

```diff
@@ -117,143 +117,67 @@ Note -
 
     paver -h
 
+Note -
+Unless otherwise mentioned, all the following commands should be run from inside lms docker container.
 
 Running Python Unit tests
 -------------------------
 
-We use `pytest`_ to run the test suite.
+We use `pytest`_ to run python tests. Pytest is a testing framework for python and should be your goto for local python unit testing.
```

Comment:
s/python/Python/g

Reply:
what is this format you use? I'm assuming it means switch python to Python

Reply:
Yeah, it's sed (maybe also Perl) syntax. Ned, there's also an "insert a suggestion" button that will allow you to write a suggested replacement, which the committer can even batch up with other suggestions.